pkg/bitbucket/commits.go

```diff
@@ -109,6 +109,9 @@ type CommitListParams struct {
 	Until string `json:"until"`
 }
 
+// CommitList retrieves a page of commits from a given starting commit or "between" two commits. If no explicit commit is specified, the tip of the repository's default branch is assumed. commits may be identified by branch or tag name or by ID. A path may be supplied to restrict the returned commits to only those which affect that path.
+// The authenticated user must have REPO_READ permission for the specified repository to call this resource.
+// https://docs.atlassian.com/bitbucket-server/rest/7.13.0/bitbucket-rest.html#idp222
 func (c *Client) CommitList(projectKey string, repositorySlug string, params CommitListParams) (*CommitPage, error) {
```

Comment:
@michaelsauter In the case where there are hundreds of commits, do we support pagination to retrieve all of them eventually?

Reply:
Not yet - this will need to be done but we can implement when there is a need I think.